collectors/ebpf.plugin/README.md

```diff
@@ -221,8 +221,8 @@ The following options are available:
     range of IPs, or use CIDR values. The default behavior is to only collect data for private IP addresess, but this
     can be changed with the `ips` setting.
     
-By default, Netdata displays up to 500 dimensions on network viewer charts. If there are more possible dimensions, they
-will be bundled into the `other` dimension. You can increase the number of shown dimensions by changing the `maximum
+By default, Netdata displays up to 500 dimensions on network connections charts. If there are more possible dimensions, 
```

Comment:
I think you should change `network connections charts` -> `network connection charts` everywhere.

Reply:
Thank you @vlvkobal , I finished the fixes you required.